vnext/Microsoft.ReactNative/Fabric/Composition/UiaHelpers.cpp

```diff
@@ -163,6 +163,18 @@ void UpdateUiaProperty(
       spProviderSimple.get(), propId, CComVariant(oldValue.c_str()), CComVariant(newValue.c_str()));
 }
 
+long GetLiveSetting(const std::string &liveRegion) noexcept {
+  if (liveRegion == "none"){
+    return LiveSetting::Off;
```

Comment:
It seems redundant to have this block if the default return value is `LiveSetting::Off`.